README.md

```diff
@@ -325,21 +325,22 @@ You can also install the extension without access to the internet by following t
 
 ## Configuration
 
-The Puppet VSCode extension can use the PDK or the Puppet Agent to provide advanced features, which can be set using two configuration options: `puppet.installType` and `puppet.installDirectory`.
+### Puppet Source Configuration
 
-- By default the `puppet.installType` is set to `agent`, and allows you to choose to use `pdk` or `agent`.
-- The `puppet.installDirectory` allows you to choose a custom path to either a PDK install or a Puppet-Agent install, and must match the correct setting in `puppet.installType`.
+The Puppet VSCode extension can use either the `PDK` or the `Puppet Agent` as a source to provide advanced features. Which source it uses, and what location to find them in, can be set using two configuration options: `puppet.installType` and `puppet.installDirectory`.
 
-If no settings are chosen, the extension will attempt to use the Puppet Agent, to force it to use the PDK you have to select `pdk` for `puppet.installType`.
+- By default the `puppet.installType` is set to `auto`, and allows you to choose to use `auto`, `pdk` or `agent`.
```

Comment:
Reads better to list the options and then say the default, rather than the opposite e.g.

`- The `xxxxx` allows you to choose to use `auto`, `pdk` or `agent` and is set to `auto` by default`

Reply:
👍